src/index.js

```diff
@@ -0,0 +1,317 @@
+
+// Search Bar For Desktop
+const isDesktop = window.innerWidth > 768;
+window.onresize = () => {
+  if (isDesktop && window.innerWidth <= 768) {
+    window.location.reload();
+  }
+
+  if (!isDesktop && window.innerWidth > 768) {
+    window.location.reload();
+  }
+};
+
+if (window.innerWidth > 768) {
+  searchBarContainer.classList.remove('hide');
+  menuIcon.classList.remove('hide');
+  const menuText = document.createElement('span');
+  menuText.style.fontSize = '1.25rem';
+  menuText.textContent += 'Menu';
+  menuIcon.append(menuText);
+  searchCloseBtn.classList.add('hide');
+  searchIcon.classList.add('hide');
+
+  // Add Search Icon
+  const searchIconDesktopBtn = document.createElement('button');
+  searchIconDesktopBtn.setAttribute('type', 'button');
+  searchIconDesktopBtn.classList.add('searchIconDesktopBtn');
+  const searchIconDesktopSpan = document.createElement('span');
+  searchIconDesktopSpan.classList.add('material-icons-round', 'icons');
+  searchIconDesktopSpan.textContent = 'search';
+
+  searchIconDesktopBtn.append(searchIconDesktopSpan);
+  header.append(searchIconDesktopBtn);
+}
+
+// Get Data from TVMAZE API
+const rootUrl = 'https://api.tvmaze.com/singlesearch/shows?q=';
+const searchResults = document.querySelector('h2');
+let query = '';
+
+// Update Likes
+const updateLikes = async () => {
+  const response = await getLikes();
+  document.querySelectorAll('.starCount').forEach((button) => {
+    for (let i = 0; i < response.length; i += 1) {
+      if (response[i].item_id === Number(button.id)) {
+        button.lastChild.textContent = response[i].likes;
+      }
+    }
+  });
+};
+
+// Display Cards Dynamically
+const cards = document.querySelector('.cards');
+const createElement = async (requestURL) => {
+  cards.innerHTML = '';
+  await getData(requestURL)
+    .then((data) => {
+      let searchCount = 0;
+      // eslint-disable-next-line no-underscore-dangle
+      const dataArray = data._embedded.episodes;
+      dataArray.forEach((el) => {
+        const div = document.createElement('div');
+        div.classList.add('cardItem');
+        const divImg = document.createElement('div');
+        divImg.classList.add('cardImg');
+        divImg.style.backgroundImage = `url(${el.image.original})`;
+        const h2 = document.createElement('h2');
+        h2.classList.add('cardName');
+        h2.textContent = `S${el.season}E${el.number} ${el.name}`;
+        const details = document.createElement('p');
+        details.classList.add('cardDetails');
+        details.innerHTML = `Plot Summary: <br>${el.summary}`;
+        const h3 = document.createElement('h3');
+        h3.classList.add('cardRuntime');
+        h3.textContent = `Runtime: ${el.runtime} mins Rating: ${el.rating.average}`;
+
+        const starContainer = document.createElement('div');
+        starContainer.classList.add('starContainer');
+
+        const starRate = document.createElement('span');
```

Comment:
- [ ] [optional] It would have been nice to have only one star for liking the movie, two stars are confusing, esp to know which one to click to add a like.